packages/react-dom-bindings/src/shared/ReactDOMUnknownPropertyHook.js

```diff
@@ -311,17 +304,9 @@ function validateProperty(tagName, name, value, eventRegistry) {
             case 'reversed':
             case 'scoped':
             case 'seamless':
-            case 'itemScope': {
-              break;
-            }
+            case 'itemScope':
             case 'inert': {
-              if (enableNewBooleanProps) {
-                break;
-              }
-            }
-            // fallthrough for new boolean props without the flag on
-            default: {
-              return true;
```

Comment:
Do we need to keep the `default` case? I wonder if the comment is only referring to the switch fallthrough lint bypass.

Reply:
Comment was only for lint bypass.